Rollbar/Rollbar.cs

```diff
@@ -53,7 +53,8 @@ public static void PersonData(Func<Person> personFunc)
             payload.Data.GuidUuid = guid;
             payload.Data.Person = _personFunc?.Invoke();
 
```

Comment:
Error		SA1500 : CSharp.Layout : If a statement spans multiple lines, the opening curly bracket must be placed on its own line.